lib/introspector.js

```diff
@@ -152,32 +153,52 @@ const parseParameters = lines => {
   return result;
 };
 
-const parseRest = lines => {
-  if (lines.length === 0) return [];
+const splitRest = lines => {
   const result = [];
+  let temp = [];
   for (let i = 0; i < lines.length; i++) {
     if (isNamedLine(lines[i])) {
-      const [name, cmt] = common.section(lines[i], ':');
-      let types = [];
-      let comment = cmt;
-      let nonStandardTypes = [];
-      if (name === 'Returns' || name === 'Throws') {
-        [types, nonStandardTypes, comment] = parseTypes(comment.trimLeft());
-      }
-      result.push({ name, types, nonStandardTypes, comment });
+      if (i !== 0) result.push(temp);
+      temp = [ lines[i] ];
     } else {
-      result[result.length - 1].comment += '\n' + lines[i];
+      temp.push(lines[i]);
     }
   }
```

Comment:
```js
const result = [[]];
for (let i = 0; i < lines.length; i++) {
  if (isNamedLine(lines[i]) {
    result.push([lines[i]]);
  } else {
    result[result.length - 1].push(lines[i])
  }
}
```
IIUC?

Reply:
Yes, but what is the point in `const result = [[]];`? It will change the behaviour. Also, WDYT about `for...of`?
```js
const result = [];
for (const line of lines) {
  if (isNamedLine(line) {
    result.push([line]);
  } else {
    result[result.length - 1].push(line);
  }
}
```